src/hw_isolation_record/manager.cpp

```diff
@@ -515,6 +515,14 @@ void Manager::createEntryForRecord(const openpower_guard::GuardRecord& record,
             return;
         }
 
+        // We have a valid case where errolog path might be empty due to PEL
+        // deletion
+        std::string strBmcErrorLogPath{};
```

Comment:
Looks like you didn't use it anywhere...
JS: you could do this like below
```
std::string errorLogObjPath;
if (!bmcErrorLogPath.has_value())
{
   // Add a error trace to indicate "the given pel id [xxxx] is not found in the BMC error logs"
   if (!isRestorePath)
   {
      return;
   }
}
else
{
errorLogObjPath = bmcErrorLogPath->str;
}

// Use "errorLogObjPath" instead of "bmcErrorLogPath->str" in line number 533
```

and please include journal trace for success and failure case in the commit message.